### ISSUE TITLE
form.choice disabled option label contrast failure

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1169]
Where choice options are disabled as radios or checkboxes, disabled option labels are flagged by SiteImprove as failing colour contrast:

![image](https://user-images.githubusercontent.com/756393/77052241-6f4c4f80-69c4-11ea-97bf-1e939cc11f34.png)

Investigate if this is a false failure. Does the wrapping label require `aria-disabled="true"`?

**Comment by Stanton** [comment]
`aria-disabled="true"` on the wrapping label still results in SI complaining

**Comment by Stanton** [comment]
![Screenshot 2020-03-23 at 12 35 25](https://user-images.githubusercontent.com/18653/77317283-cfaff980-6d02-11ea-8bd3-479449d2ed0c.png)

Same with regular links, or anything not using the `disabled` attribute.

**Comment by Stanton** [comment]
For radio/checkboxes we could rely on disabled styling the input only, or also add a 'disabled' text to the label, this wouldn't improve the situation on links unless we rolled that out there too...

![Screenshot 2020-03-23 at 12 34 07](https://user-images.githubusercontent.com/18653/77317496-36351780-6d03-11ea-9232-9317047def4e.png)

**Comment by Stanton** [comment]
If we use `opacity` instead of a lighter `color`, SI can't see it. I think as long as the elements are appropriately marked up as disabled for AT and aren't focusable, I believe this change would comply with WCAG which doesn't require disabled elements to meet contrast requirements....

**Comment by jamesjacobs** [comment]
Fixed by #1177
